core/platform/auth/firebase_auth_services_test.py

```
@@ -184,8 +184,11 @@ def create_session_cookie(
         return session_cookie
 
     def create_user(
-        self, uid: str, email: Optional[str] = None, disabled: bool = False
-    ) -> str:
+        self,
+        uid: Optional[str] = None,
+        email: Optional[str] = None,
+        disabled: bool = False,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)